pkg/doc/verifiable/common.go

```diff
@@ -92,6 +92,8 @@ func (r *DIDKeyResolver) resolvePublicKey(issuerDID, keyID string) (*verifier.Pu
 
 	methods := doc.VerificationMethods()
 	for _, verificationMethods := range methods {
+		// TODO verify which verificationMethod should be checked in order of priority. Looping through the list isn't
```

Comment:
source?

Reply:
currently this logic simply loops through the list verification methods. This works right now because the list of verifications all have the same key.  And there is only 1 key in each.

I believe verifying proof should use the `authentication` method, right? I don't see this logic anywhere.